data/json/items/melee/swords_and_blades.json

```
@@ -322,6 +480,59 @@
     },
     "extend": { "flags": [ "PUMP_RAIL_COMPATIBLE" ] }
   },
+  {
+    "id": "knife_combat_ti",
+    "copy-from": "knife_combat",
+    "type": "TOOL",
+    "name": { "str": "MPK-TI alloy knife", "str_pl": "MPK-TI alloy knives" },
+    "looks_like": "knife_combat",
+    "description": "A tempered titanium alloy combat knife, originally designed for naval mine clearance teams.  Durable, non-conductive, lighter than steel, and can even be mounted as a rifle bayonet.  The Navy once claimed these knives had never broken during a mission; whether this reputation holds up after the end of the world remains to be seen.",
+    "//": "based on: https://missionknives.com/products/mpk-12",
+    "material": [ "qt_ti_alloy", "thermo_resin" ],
+    "color": "light_gray",
+    "repairs_with": [ "qt_ti_alloy" ],
+    "weight": "280 g",
+    "price": 40000,
+    "price_postapoc": 2500,
+    "delete": { "flags": [ "CONDUCTIVE" ] },
+    "gunmod_data": {
+      "location": "underbarrel",
+      "mod_targets": [ "shotgun", "rifle", "smg", "crossbow" ],
+      "mode_modifier": [ [ "REACH", "bayonet", 2, [ "MELEE", "REACH_ATTACK" ] ] ],
+      "install_time": "5 s"
+    },
+    "qualities": [ [ "CUT", 2 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 22 ] ],
+    "thrown_damage": [ { "damage_type": "stab", "amount": 16 } ],
+    "weapon_category": [ "KNIVES" ],
+    "melee_damage": { "bash": 4, "stab": 27 }
+  },
+  {
+    "id": "tanto_tool_ti",
+    "copy-from": "tanto",
+    "type": "TOOL",
+    "name": { "str_sp": "MDK-TI tactical tanto" },
+    "looks_like": "tanto",
+    "description": "A tempered titanium alloy straight blade designed for Navy EOD teams, and patterned after the Japanese tanto.  Its rugged, non-conductive construction makes it a serviceable tool for prying, or digging through soil without setting off magnetic mines.  It's an effective weapon as well, whether cutting durable materials or rotten flesh, and can be attached to weapons as a long bayonet.",
+    "//": "based on: https://missionknives.com/products/mdk",
+    "material": [ "qt_ti_alloy", "thermo_resin" ],
+    "color": "light_gray",
+    "repairs_with": [ "qt_ti_alloy" ],
+    "weight": "330 g",
+    "volume": "300 ml",
+    "longest_side": "35 cm",
+    "price": 50000,
+    "price_postapoc": 3000,
+    "delete": { "flags": [ "CONDUCTIVE" ] },
+    "gunmod_data": {
+      "location": "underbarrel",
+      "mod_targets": [ "shotgun", "rifle", "crossbow", "launcher" ],
+      "mode_modifier": [ [ "REACH", "bayonet", 2, [ "MELEE", "REACH_ATTACK" ] ] ],
+      "install_time": "5 s"
+    },
+    "qualities": [ [ "CUT", 1 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 20 ] ],
```

```suggestion
    "qualities": [ [ "CUT", 1 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 20 ], [ "DIG", 1 ] ],
```
"digging through soil without setting off magnetic mines"

---

Thanks for catching this! I had dig 1 on it for my first draft of this PR, but must've lost it while refactoring and not noticed.

P.S., thanks for catching all the Its/It's. Trips me up all the time! ><;